GameDataParser/Files/SetPath.cs

```diff
@@ -2,10 +2,10 @@
 
 namespace GameDataParser.Files
 {
-    public class SetPath
+    public static class SetPath
```

Comment:
Any reason this is called `SetPath`? It sounds like it would be a method name, not the name of a class

Reply:
Didnt came with something else, but initially was treaten as method but i move it to a class, perhaps could be call, VariableSettings?

Reply:
Sure? I mean anything is better than `SetPath`, because the current name doesn't really make any sense at all